gdrive_sync/api.py

```diff
@@ -282,12 +282,6 @@ def process_file_result(
 @retry_on_failure
 def stream_to_s3(drive_file: DriveFile):
     """Stream a Google Drive file to S3"""
-    if drive_file.status in (
-        DriveFileStatus.UPLOAD_COMPLETE,
-        DriveFileStatus.TRANSCODING,
-        DriveFileStatus.COMPLETE,
-    ):
-        return
```

Comment:
This was added in https://github.com/mitodl/ocw-studio/pull/2569, apparently as a means of not streaming already streamed files to s3 on a gdrive sync. However, for already streamed files, we [do not call](https://github.com/mitodl/ocw-studio/blob/master/gdrive_sync/tasks.py#L165-L166) (the `if drive_file` check fails if the file is synced before and has no new changes. see [comment](https://github.com/mitodl/ocw-studio/blob/master/gdrive_sync/api.py#L141-L143) ) process_drive_file (which calls this) so I think it is safe to remove.